extensions/amp-story/0.1/amp-story-bookend.css

```diff
@@ -27,28 +27,18 @@
   right: 0 !important;
   top: 0 !important;
   z-index: 100001 !important;
-  transition: transform 0.3s ease-out !important;
+  transition: transform 0.3s cubic-bezier(0.0, 0.0, 0.2, 1) !important;
 }
 
 .i-amphtml-story-bookend.i-amphtml-hidden {
-  transition-timing-function: ease-in !important;
+  transition: transform 0.2s cubic-bezier(0.4, 0.0, 1, 1) !important;
   transform: translateY(100vh) !important;
   pointer-events: none !important;
 }
 
-.i-amphtml-story-bookend-hidden {
-  pointer-events: none !important;
```

Comment:
Why isn't this (and below) needed anymore?

Reply:
From what I see this is zombie code, the class was never set. The element is not visible anyway (``translateY(100vh)``)